documentation/applying_manifest_blocks.md

```diff
@@ -43,21 +42,37 @@ task](https://forge.puppet.com/puppetlabs/puppet_agent) to install the agent.
 
 ### Applying manifest files
 
-To apply Puppet code from an existing manifest file, use the `bolt apply`
-command and specify the absolute path to the manifest file:
+Bolt can apply a manifest from a file by passing an absolute path to the
+manifest:
 
-```shell
-$ bolt apply ~/bolt/site-modules/profiles/manifests/server.pp -t target1,target2
-```
+- _\*nix shell command_
+
+  ```shell
+  bolt apply ~/bolt/site-modules/profiles/manifests/server.pp --targets servers
+  ```
+
+- _PowerShell cmdlet_
+
+  ```powershell
+  Invoke-BoltApply -Manifest ~/bolt/site-modules/profiles/manifests/server.pp -Targets servers
```

Comment:
Is `~` the right thing to use here or is there something more idiomatic?

Reply:
Hm, it might make more sense to just use a path relative to the cwd.

Reply:
Since you can have `manifests/` inside the project now, `manifests/server.pp` might be sufficient